### PR TITLE
Add missing links to blog article

### PR DESCRIPTION
**Pull request by grabbou**
When copying meeting notes from Google Docs, I missed some of the links.

### CODE REVIEW

File: blog/2017-08-30-react-native-monthly-3.md [hunk 1]
```diff
@@ -46,14 +46,14 @@ Here are the notes from each team:
...
+- React Native enabled us to power both the iOS and Android Skype apps from the same codebase. We also want to use that codebase to power the Skype web app. To help us achieve that goal, we built and open sourced a thin layer on top of React/React Native called [ReactXP](https://microsoft.github.io/reactxp/blog/2017/04/06/introducing-reactxp.html). ReactXP provides a set of cross platform components that get mapped to React Native when targeting iOS/Android and to react-dom when targeting the web. ReactXP's goals are similar to another open source library called React Native for Web. There's a brief description of how the approaches of these libraries differ in the [ReactXP FAQ](https://microsoft.github.io/reactxp/docs/faq.html).
 
 ### Shoutem
 
-- We are continuing are efforts on improving and simplifying the developer experience when building apps using Shoutem.
+- We are continuing are efforts on improving and simplifying the developer experience when building apps using [Shoutem](https://microsoft.github.io/reactxp/docs/faq.html).
```
Comment (zrumenjak) [inline blog/2017-08-30-react-native-monthly-3.md]: @grabbou you copied the wrong like here, it should be https://shoutem.github.io/.

File: blog/2017-08-30-react-native-monthly-3.md [hunk 2]
```diff
@@ -46,14 +46,14 @@ Here are the notes from each team:
...
 - Started migrating all our apps to react-navigation, but we ended postponing this until a more stable version is released, or one of the native navigation solutions becomes stable.
-- Updating all our extensions and most of our open source libraries (animation, theme, ui) to React Native 0.47.1.
+- Updating all our [extensions](https://microsoft.github.io/reactxp/docs/faq.html) and most of our open source libraries ([animation](https://github.com/shoutem/animation), [theme](https://github.com/shoutem/theme), [ui](https://github.com/shoutem/ui)) to React Native 0.47.1.
```
Comment (zrumenjak) [inline blog/2017-08-30-react-native-monthly-3.md]: Same as above, the link for extensions should be https://github.com/shoutem/extensions

**Comment by grabbou** [comment]
Thanks, looks my copy-paste didn't work that time!

**Comment by beetlebum** [comment]
There's a typo in the text `We are continuing are efforts`

**Comment by exKAZUu** [comment]
I think
`Continuing to work with [Krzysztof](https://github.com/kmagiera/react-native-gesture-handler) on [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler).`
should be
`Continuing to work with [Krzysztof](https://github.com/kmagiera) on [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler).`
.

**Pull request closed without merging**